github.go

```diff
@@ -1,11 +1,19 @@
 package reviewdog
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Review should have comment or be unexported